data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -127,6 +137,14 @@ t_hypothesis_annotation_with_evaluation_suffix AS (
   FROM t_hypothesis_annotation_with_osf_doi AS annotation
   INNER JOIN t_hypothesis_with_source_doi_rank AS doi_rank
     ON annotation.uri = doi_rank.uri
+    AND (
+      CASE 
+        WHEN annotation.osf_preprint_version_rank IS NOT NULL
```

Comment:
For the condition I would find following easier to read:

```sql
annotation.osf_preprint_version_rank IS NULL
OR annotation.osf_preprint_version_rank = doi_rank.source_doi_rank
```

But then somehow it feels "odd" to make the default true. (I will need to think more about it)